src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Optional<Order> getByUser(User user);
```

Comment:
How we can use this method? One user could have many orders, which one do we get here?